backend/glp/service.py

```
@@ -0,0 +1,684 @@
+                guild_id=allocation.guild_id,
+                discord_id=allocation.discord_id,
+                status=GLPReservation.STATUS_RESERVED,
+            )
+            .first()
+        )
+        if reservation is None:
+            return None
+        return _claim_locked(program, reservation, allocation.user)
+
+
+def _sweep_unclaimed_reservations(program):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_sweep_unclaimed_reservations`

Add return type annotation: `None`

(ANN202)